docs/guidance/example.md

```diff
@@ -62,7 +62,7 @@ Each project object contains the following sections:
 
 This section provides contextual information about the infrastructure project, including:
 
-* a project identifier in the `id` field. To make the project identifier globally unique, a project identifier prefix must be added to the local identifier for the project. Project identifier prefixes are assigned by the OC4IDS helpdesk. For more information on project identifiers, refer to the [project identifiers guidance](https://github.com/open-contracting/infrastructure/pull/159/identifiers.md).
+* a project identifier in the `id` field. To make the project identifier globally unique, a project identifier prefix must be added to the local identifier for the project. Project identifier prefixes are assigned by the OC4IDS helpdesk. For more information on project identifiers, refer to the [project identifiers guidance](https://standard.open-contracting.org/infrastructure/latest/en/guidance/identifiers/#globally-unique-project-identifiers).
 
```

Comment:
I assume you've used an absolute link because you couldn't get a relative link to a HTML bookmark to work.

I had the same problem in https://github.com/open-contracting/infrastructure/pull/166 so I've asked @kindly @Bjwebb whether this is possible, otherwise I think it would be better to use the relative link to the parent page, so that it works in translated versions etc.

Reply:
Thanks @duncandewhurst, I've now replaced that absolute link for a relative one (../../../../guidance/identifiers/#globally-unique-project-identifiers)